### PR TITLE
feat(dwp): converge protocol on DWP/GC naming

### PR DESCRIPTION
**Pull request by hacknlove**
## Summary
- align Go + JS implementations on DWP/GC naming with `.dwp` paths and `dwp-*` trailers (no legacy fallback)
- update docs/assets and workflow pack to reflect DWP/GC protocol details
- add/adjust tests for DWP remote parsing, events output, and last-wins state

## Testing
- go test ./...
- npm test
- npm run build (docs) — warning: existing Starlight duplicate id warnings

### CODE REVIEW

**Comment by hacknlove** [comment]
### Commit review: 282644c — feat(dwp): converge protocol on DWP/GC naming

**Intent:** full convergence on DWP/GC naming + `.dwp/*` layout + `dwp-*` trailers (no legacy fallback). ✅

**What changed (high level):**
- Removes legacy `.aynig/*` assets (`.aynig/CONTRACT.md`, `.aynig/command/clean`) and updates contract/docs to DWP/GC.
- Aligns JS + Go implementations on `dwp-state`/`dwp-*` trailers and `.dwp` command resolution.
- Updates ops-workflow-pack + docs pages to reference DWP/GC consistently.
- Adds/adjusts tests (Go + JS) around events output, state parsing (last-wins), and remote parsing.

**Verification I ran locally:**
- `cd js && npm test` ✅
- `cd go && go test ./...` ✅

**Notes / watch-outs:**
- Big sweeping rename (74 files). Given “no back-compat”, deleting `.aynig/*` is correct; just calling out that it’s intentionally breaking.
- Docs build warnings about duplicate IDs are pre-existing per PR description; not a blocker for this change itself.

**Review by hacknlove — COMMENTED** [review]
Reviewed commit 282644c. Ran JS and Go test suites successfully. PR consistently converges on DWP/GC naming (dwp-* trailers + .dwp layout) with no legacy fallback.

File: js/commands/status.js [hunk 1]
```diff
@@ -74,7 +74,6 @@ async function action(options) {
     if (shouldResolveCommand && commandState && commandState !== 'working') {
       const roleName =
         String(options.role || '').trim() ||
-        String(process.env.DWP_ROLE || '').trim() ||
         String(process.env.AYNIG_ROLE || '').trim();
```
Comment (hacknlove) [inline js/commands/status.js:77]: Role env var: since we are fully converged on DWP/GC naming, consider using `DWP_ROLE` here (or supporting both `DWP_ROLE` and `AYNIG_ROLE` for a transition). Right now status only checks `AYNIG_ROLE`.

File: docs/src/content/docs/commands/environment.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ Common variables:
 - `AYNIG_BODY` — the commit message body (prompt)
 - `AYNIG_COMMIT_HASH` — the triggering commit hash
 - `AYNIG_LOG_LEVEL` — resolved log level for the run/branch
-- `AYNIG_ROLE` — selects `.aynig/roles/<role>/command` when set
+- `AYNIG_ROLE` — selects `.dwp/roles/<role>/command` when set
```
Comment (hacknlove) [inline docs/src/content/docs/commands/environment.md:13]: Docs: this page still mentions `AYNIG_ROLE` for role-based command resolution. If we want full DWP/GC naming, either rename to `DWP_ROLE` or document both env vars + precedence.

Comment (hacknlove) [reply docs/src/content/docs/commands/environment.md:13]: environmental variables are implementation details, it's not for the protocol to decide how each host decide what role it manages

File: js/commands/status.js (hunk 1 above)
Comment (hacknlove) [reply js/commands/status.js:77]: as mentioned before, the env variables are outside the protocol, it's ok to be AYNIG_ROLE

File: docs/src/content/docs/operate/runbooks.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ If a command fails, AYNIG does not retry automatically. Your workflow should dec
 
 Common patterns:
 
-- Emit a new commit with a failure state (e.g. `aynig-state: failed`)
+- Emit a new commit with a failure state (e.g. `dwp-state: failed`)
 - Increment an attempt trailer (`aynig-attempt:`) and re-emit the same state
```
[the remark this answers is not in the document]
Comment (hacknlove) [reply docs/src/content/docs/operate/runbooks.md]: Updated to dwp-attempt in runbooks (docs consistency with trailer conventions).

File: docs/src/content/docs/operate/concurrency-patterns.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ AYNIG provides mutual exclusion per branch. Concurrency patterns are implemented
 
 Examples (conceptual):
 
-- Fan-out: create N branches each with its own `aynig-state`
+- Fan-out: create N branches each with its own `dwp-state`
 - Fan-in: a coordinator branch waits for results and merges or references outputs
 
 Use correlation trailers (e.g. `aynig-correlation-id`) to relate work across branches.
```
[the remark this answers is not in the document]
Comment (hacknlove) [reply docs/src/content/docs/operate/concurrency-patterns.md]: Updated to dwp-correlation-id for consistency with trailer conventions.

File: docs/src/content/docs/operate/retries.md [hunk 5]
```diff
@@ -10,7 +10,7 @@ A simple convention:
 - Use trailers like:
   - `aynig-attempt: 1`
   - `aynig-max-attempts: 3`
```
[the remark this answers is not in the document]
Comment (hacknlove) [reply docs/src/content/docs/operate/retries.md]: Updated retry trailer examples to dwp-attempt / dwp-max-attempts.

File: ops-workflow-pack/command/stalled.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
 
 Your goal is to decide the next state.
 
 Instructions:
-1) Read `.aynig/COMMANDS.md` to learn:
+1) Read `.dwp/COMMANDS.md` to learn:
    - what `$AYNIG_ORIGIN_STATE` means and its expected behavior
    - other available states and their instructions
-2) Find the initial commit of this job: the closest commit with `aynig-state: $AYNIG_ORIGIN_STATE`.
+2) Find the initial commit of this job: the closest commit with `dwp-state: $AYNIG_ORIGIN_STATE`.
```
[the remark this answers is not in the document]
Comment (hacknlove) [reply ops-workflow-pack/command/stalled.md]: Updated stalled runbook: removed undefined AYNIG_ORIGIN_STATE and now instructs using dwp-stalled-run to find the last working commit and read dwp-origin-state. Also preserved dwp-origin-state on stalled commits in code.

File: ops-workflow-pack/README.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
 This folder contains an optional workflow pack you can install into any repo.
 
-Included commands live under `.aynig/command/` and are intentionally minimal.
+Included commands live under `.dwp/command/` and are intentionally minimal.
 Use them as starting points and adapt to your own operating patterns.
```
[the remark this answers is not in the document]
Comment (hacknlove) [reply ops-workflow-pack/README.md]: Clarified workflow pack layout: commands live in ops-workflow-pack/command/ and install into .dwp/command/.

File: go/internal/commands/install.go [hunk 8]
```diff
@@ -44,16 +44,16 @@ func Install(repo string, ref string, subfolder string) error {
 		if subfolder != "" {
 			subfolderInfo = " in " + subfolder
 		}
-		return fmt.Errorf("Error: No .aynig directory found in %s%s%s", repo, refInfo, subfolderInfo)
+		return fmt.Errorf("Error: No .dwp directory found in %s%s%s", repo, refInfo, subfolderInfo)
 	}
```
[the remark this answers is not in the document]
Comment (hacknlove) [reply go/internal/commands/install.go:51]: Adjusted install error messaging to distinguish missing subfolder vs missing .dwp (in both Go and JS).

File: go/internal/statex/state_commit.go [hunk 9]
```diff
@@ -47,15 +47,13 @@ func VerifyHeadStateTrailer(dir string) error {
...
 		return errors.New("Invalid trailer block: trailers must be contiguous at end of message")
 	}
-	if len(stateValues) > 1 {
-		return errors.New("Invalid trailer block: multiple aynig-state trailers")
-	}
-	if strings.TrimSpace(stateValues[0]) == "" {
-		return errors.New("Invalid trailer block: empty aynig-state trailer")
+	last := stateValues[len(stateValues)-1]
+	if strings.TrimSpace(last) == "" {
+		return errors.New("Invalid trailer block: empty dwp-state trailer")
```
[the remark this answers is not in the document]
Comment (hacknlove) [reply go/internal/statex/state_commit.go:56]: Updated  to return a more specific  error when trailers parse cleanly but no state is present, while keeping the existing  behavior. I kept the implementation KISS: no extra footer heuristics, and added end-to-end Go tests for valid state, last-wins, missing state, and empty state.

[the remark this answers is not in the document]
Comment (hacknlove) [reply go/internal/statex/state_commit.go:56]: I updated VerifyHeadStateTrailer to return a more specific 'missing dwp-state trailer' error when trailers parse cleanly but no state is present, while keeping the existing 'empty dwp-state trailer' behavior. I kept the implementation KISS: no extra footer heuristics, and added end-to-end Go tests for valid state, last-wins, missing state, and empty state.
